fastlane/metadata/fr-FR/keywords.txt

```diff
@@ -1 +1 @@
-notes,texte,planificateur,journal,synchronisation,choses à faire,cloud,notebook,simple,bloc-notes
\ No newline at end of file
+note,markdown,journal,synchronisation,àfaire,liste,cloud,notebook,tâches,notepad,étiquette,écrire
```

Comment:
I notice the French translators only included the singular `note` in their translation but not the plural `notes` one that the original copy has. They also didn't respect the order of some of the keywords near the end of the list


```diff
-note,markdown,journal,synchronisation,àfaire,liste,cloud,notebook,tâches,notepad,étiquette,écrire
+notes,note,markdown,journal,synchronisation,àfaire,liste,cloud,notebook,simple,notepad,étiquette
```

I wonder if it's worth a comment in the +i18n post to make them fix it (again) for the next cycle. Especially because I only checked French but I'm wondering if translators for other languages might have not respected the order of the keywords (or which ones were allowed to be removed) either…

otoh the keywords that ends up the most searched in English are not the same as the ones being the most searched in French (I very highly doubt that anyone would search "àfaire" especially because these are supposed to be 2 words "à faire" anyway, and not a term we'd use as a noun like we'd use "todo" in English), so not sure that matters that much…?

Reply:
@AliSoftware It's up to you whether you'd like to post a comment about this. Alternatively, you can make a suggestion for the French translation. I doubt any concrete action could be taken for other languages by just sending it back 🤷

Reply:
Added suggestion in GP for FR keywords.